CMakeLists.txt

```diff
@@ -1882,10 +1882,10 @@ if(BROADCAST)
       add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/lib/libshout")
     endif()
   else()
-    find_package(Shout)
     if(NOT Shout_FOUND)
         message(FATAL_ERROR "Live Broadcasting support requires the libshout and its development headers.")
     endif()
+    target_link_libraries(mixxx-lib PUBLIC Shout::Shout)
```

Comment:
Now libshout is only linked on Windows/OSX.
Linux/BSD never link libshout, neither the system one nor the version from the lib directory.